docs/source/conf.py

```diff
@@ -1,8 +1,8 @@
+# Copyright (c) 2018-2023, NVIDIA CORPORATION.
```

Comment:
@galipremsagar The copyright must be below the hashbang `#!/usr/bin/env python3` in order for this file to be properly executable.

The `# -*- coding: utf-8 -*-` line is extraneous and no longer needed in modern Python. Please remove that.